.github/ISSUE_TEMPLATE/04_user_support.yml

```diff
@@ -0,0 +1,27 @@
+name: User Support
+description: Do you need assistance with admiral?
+title: "Guidance and support on how to use admiral"
+labels: ["user support"]
+assignees:
+  - octocat
+body:
+  - type: markdown
+    attributes:
+      value: To get immediate help check out and join our slack channel - [admiral](https://app.slack.com/client/T028PB489D3/C02M8KN8269/thread/C028SJ83KM1-1643222485.137579)
```

Comment:
I would remove the thread part of the link, i.e., use `https://app.slack.com/client/T028PB489D3/C02M8KN8269`

Reply:
removed